src/client.ts

```diff
@@ -0,0 +1,34 @@
+import axios, { InternalAxiosRequestConfig, AxiosResponse } from 'axios';
```

Comment:
It seems weird to have to use `InternalAxiosRequestConfig` which I assume was intended to be internal (despite being exported) rather than `AxiosRequestConfig` but the latter doesn't seem to work. :shrug:

Reply:
I see [the issue was already reported](https://github.com/axios/axios/issues/5573). https://traveling-coderman.net/code/node-architecture/logging-http-requests/ shows using `AxiosRequestConfig` which must have worked in the past.